front/src/components/molecules/Dropdown.tsx

```diff
@@ -15,7 +16,7 @@ export const Dropdown = ({ options, selected, onSelect }: DropdownProps) => {
         <Listbox value={selected} onChange={onSelect}>
             <div className="relative w-full">
                 <Listbox.Button className="relative w-full border-2 border-gray-300 rounded-lg bg-white h-10 py-2 pl-3 pr-10 text-left hover:bg-gray-100 focus:border-theodo-blue focus:outline-none focus-visible:border-theodo-blue focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75 focus-visible:ring-offset-2 focus-visible:ring-offset-theodo-grey sm:text-sm">
```

Comment:
scary 😂

Reply:
Yep! That is because of the interaction styles...

Reply:
I wonder if some of the styles can be combined? Did tailwind ever merge the `hover:( ... )` group syntax?

Reply:
@Lerri-Cofannos